private/Producers/FbxProducer/FbxProducerImpl.cpp

```diff
@@ -954,64 +958,126 @@ cd::BoneID FbxProducerImpl::AddBone(const fbxsdk::FbxNode* pSDKNode, cd::BoneID
 
 void FbxProducerImpl::ProcessAnimation(fbxsdk::FbxScene* scene, cd::SceneDatabase* pSceneDatabase)
 {	
+	// Locates all skeleton nodes in the fbx scene. Some might be nullptr.
+	std::vector<FbxNode*> bones;
+	for (int i = 0; i < pSceneDatabase->GetBoneCount(); i++) 
+	{
+		const char* jointName = pSceneDatabase->GetBone(i).GetName();
```

Comment:
```suggestion
		const char* pJointName = pSceneDatabase->GetBone(i).GetName();
```
We used to add prefix "p" for pointer type variables.